prime-router/src/main/kotlin/FakeReport.kt

```diff
@@ -44,6 +49,7 @@ class FakeDataService : Logging {
                 element.nameContains("name_of_school") -> randomChoice("", context.schoolName)
                 element.nameContains("reference_range") -> randomChoice("", "Normal", "Abnormal", "Negative")
                 element.nameContains("result_format") -> "CWE"
+                element.nameContains("site_of_care") -> siteOfCare[(0..18).random()]
```

Comment:
@oslynn instead of calling random on the range object, you can use the built-in method `randomChoice` as we are using above. Please change it to use that.

Reply:
Done.